app/metrics.yaml

```diff
@@ -2030,6 +2030,20 @@ browser.search:
       notification_emails:
         - fenix-core@mozilla.com
       expires: "2020-09-01"
+    in_content:
+      type: labeled_counter
+      description: >
+        Records the type of interaction a user has on SERP pages.
```

Comment:
Can you include the types of data that this could be, or provide an example? That way, if someone from data science is looking at our documentation, they have a very clear idea of what kind of data this looks like.

Reply:
Since this PR was merged already, I'll add a new task with the docs change